.github/workflows/token-detection.yml

```diff
@@ -5,7 +5,7 @@ on:
 
 jobs:
   detect:
-    runs-on: [self-hosted, linux]
+    runs-on: [ubuntu-18.04]
```

Comment:
This repository is public (you can access without GitHub login, but the allowed IP list is still active) so we cannot use self-hosted runner by default.